zenrock/assetlist.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "zenrock",
+    "assets": [
+      {
+        "description": "The native token of zenrock",
+        "denom_units": [
+          {
+            "denom": "urock",
+            "exponent": 0,
+            "aliases": []
+          },
+          {
+            "denom": "rock",
+            "exponent": 6,
+            "aliases": []
+          }
+        ],
+        "base": "urock",
+        "name": "Zenrock",
+        "display": "rock",
+        "symbol": "ROCK",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/zenrock/images/zenrock.png"
+        },
+        "keywords": [
+          "mpc"
+        ],
+        "images": [
+          {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/zenrock/images/zenrock.png"
```

Comment:
Why keep so much padding on the png (which will make it appear very tiny) and register as separate images? Might as well remove the padding to make the image appear correctly and then it will visually match and they can be registered as two formats for the same image.

Reply:
We should still register them as a single image, now